lib/internal/freeze_intrinsics.js

```diff
@@ -135,7 +135,7 @@ const {
 
 module.exports = function() {
   const { Console } = require('internal/console/constructor');
-  const { console } = require('internal/console/global');
```

Comment:
Amazing find. Out of curiosity, how did you realise this and how did it not fail all this time?

Reply:
> how did you realise this

In a different PR the linter was telling me to do something, and when I did it, it didn't work.

> how did it not fail all this time?

The existing code works fine with freezing `undefined` and my guess is that no one is trying to do this in real life.

Reply:
Good way to start 2023 👍